tests/bam/bam_pb.robot

```diff
@@ -860,7 +860,7 @@ BA_RATIO_NUMBER_BA_4_SERVICE
 
 
 BA_RATIO_PERCENT_BA_4_SERVICE
-	[Documentation]	With bbdo version 3.0.1, a BA of type 'ratio number' with 4 serv
+	[Documentation]	With bbdo version 3.0.1, a BA of type 'ratio percent' with 4 serv and from the persent of the services linked to the BA, it should change it status based on the percent of the each state o the linked services.
```

Comment:
a ratio percent ba is almost the same as a ratio number. The difference is that it is defined with percents. For example, if more than 50% of the kpi are critical, we can decide the ba to be warning...